src/GafferScene/IECoreScenePreview/CapturingRenderer.cpp

```diff
@@ -108,6 +109,17 @@ Renderer::ObjectInterfacePtr CapturingRenderer::object( const std::string &name,
 {
 	checkPaused();
 
+	// To facilitate the testing of code that handles the return from the various object methods of
+	// a renderer, we return null if the `cr:unrenderable` attribute is set to true.
+	if( const auto attr = dynamic_cast<const CapturingRenderer::CapturedAttributes *>( attributes ) )
```

Comment:
@johnhaddon I went for attr instead of a primvar as it allows us to test `Renderer::camera` too.